test/run.sh

```diff
@@ -47,14 +49,20 @@ if [ $# -eq 0 ]; then
 	exit 1
 fi
 
-# load config lists
-# contains:
-#   globalTests
-#   testAlias
-#   imageTests
-#   globalExcludeTests
+# declare configuration variables
+declare -a globalTests=()
+declare -A testAlias=()
+declare -A imageTests=()
+declare -A globalExcludeTests=()
+
+# load the default config
 . "$dir/config.sh"
```

Comment:
Hmm yeah, interesting thoughts.  I think you're currently the "state of the
art" for tests outside our repo, so as long as the logic doesn't get _too_
hairy I'm +1 for whatever makes your life easier using our framework! :+1: